src/types.ts

```diff
@@ -321,13 +332,13 @@ export interface PaymentResponse
 }
 
 export interface Allowance {
-  createdAt: string;
```

Comment:
bumi and me just talked about this..
```
createdAt: string;
lastPaymentAt: string;
lnurlAuth: string;
```
This is how it's saved in the index-db of most users we assume.
So changing this to "correct" types looks great and works in this scenario but in reality this might break because i.e. `lnurlAuth` will be `"true"` and not `true`.  

We should first make sure we cover the real world types which are being used currently before we change them. If we change them we might also need to migrate the index-db for the users.

Is this understandable? You think this is an issue? Happy to talk via Slack.

Reply:
Got it! Reverting the changes,
> saved in the index-db of most users we assume

didn't know this can be a breaking change!

Reply:
I did not find code that writes the createdAt so far. https://github.com/getAlby/lightning-browser-extension/blob/master/src/extension/background-script/actions/allowances/add.js#L23-L31

ideally the timestamp is a number (like lastPaymentAt)